### PR TITLE
Fix URL for Dev UI es-shims

### PR DESCRIPTION
**Pull request by phillip-kruger**
Small fix for Dev UI

### CODE REVIEW

File: extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui-templates/build-time/index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link rel="shortcut icon" type="image/png" href="{contextRoot}favicon.ico">
 
-        <script async src="{nonApplicationRoot}_static/es-module-shims/dist/es-module-shims.js"></script>
+        <script async src="{nonApplicationRoot}_static/es-module-shims/1.8.0/dist/es-module-shims.js"></script>
```
Comment (gsmet) [inline extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui-templates/build-time/index.html:8]: Isn't this extremely brittle? What if it is updated?

Comment (phillip-kruger) [reply extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui-templates/build-time/index.html:8]: This is a bit of a corner case, as this is the thing that make importmaps work when the browser does not support importmaps, so I can not use the importmap to reference this. I can probably look into finding the version from the pom and pass that along in something like `{es-modele-version}`. Let me investigate.

**Review by cescoffier — CHANGES_REQUESTED** [review]
I agree with @gsmet - avoiding a hardcoded version in our codebase would be much better. Do you think it's possible to extract the resolved version and use that one instead?

**Comment by phillip-kruger** [comment]
Yes that should be possible, I'll have a look

File: extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui-templates/build-time/index.html (hunk 1 above)
Comment (phillip-kruger) [reply extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui-templates/build-time/index.html]: This is done

**Review by cescoffier — APPROVED** [review]

**Pull request merged**